.github/workflows/easy-cp.yml

```diff
@@ -0,0 +1,81 @@
+# Copyright 2023 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+name: Cherry-pick Labeled PR to Release Branch
+
+on:
+  pull_request:
+    branches: master
+    types: [labeled]
+
+permissions: write-all
+
+jobs:
+  cherrypick_to_release:
+    name: cherrypick_to_release
+    runs-on: ubuntu-latest
+    if: |
+      (github.event.label.name == format('cp{0} beta', ':') || github.event.label.name == format('cp{0} stable', ':')) &&
+      (github.event.pull_request.merged == true)
+    steps:
+      - name: Get Release Channel
+        run: |
+          echo "CHANNEL=$(echo ${{ github.event.label.name }} | cut -d ':' -f 2 | xargs)" >> $GITHUB_ENV
+      - name: Get Release Candidate Branch
+        run: |
+          curl -o version.json https://raw.githubusercontent.com/flutter/flutter/$CHANNEL/bin/internal/release-candidate-branch.version
```

Comment:
Is it possible to store this in memory instead of writing a file?

```sh
RELEASE_BRANCH=$(curl https://raw.githubusercontent.com/flutter/flutter/$CHANNEL/bin/internal/release-candidate-branch.version)
```

Reply:
Sure, changed from file to variable